mock-api/server.js

```diff
@@ -24,6 +24,13 @@ server.post('/comments', function (req, res, next) {
   next();
 });
 
+// serve commodity route with path resembling /api/v1/trade/commodity
+server.use(
+  jsonServer.rewriter({
+    '/api/v1/trade/commodity': '/commodity',
+  }),
+);
```

Comment:
You should be able to do this within the routes config, with something like this - 

```
{
  "/api/v1/trade/*": "/$1",
  "/api/*": "/$1",
  "/:resource/:id/show": "/:resource/:id",
  "/posts/:category": "/posts?category=:category"
}
```

Keep the more specific ones above others

Reply:
fixed: I simply removed this rewriter entry. Since we've already defined in the routes config `"/api/*": "/$1"` json-server seems to rewrite any hits to 'api/v1/trade/commodity' automatically to 'commodity' object in the db.json file.

It seems to be working for me, though I feel it shouldn't. Not a fan of their syntax here, though the module is great.